src/components/Helpers/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { SvgIcon, SvgIconInfo } from "./../SvgIcon";
+import ReactTooltip from "react-tooltip";
+
+interface ToolTipProps {
+  id: string;
+  getContent: any;
+}
+
+export const Tooltip = ({ id, getContent }: ToolTipProps) => {
```

Comment:
```suggestion
export const TooltipIcon = ({ getContent }: ToolTipProps) => {
```

Consider calling it a TooltipIcon and organise with the icons instead of it's own folder called Tooltip

Reply:
Also, from the user of Tooltip, it doesnt make any sense for me to provide a id if the id is used internally. Consider generating a unique id, see the hooks solution (second ans): https://stackoverflow.com/questions/29420835/how-to-generate-unique-ids-for-form-labels-in-react

Reply:
very nice ok